views/database_search.php

```diff
@@ -0,0 +1,38 @@
+<?php
+authorizedPage();
+include('header.php');
+?>
+
+    <div class="col-sm-8">
+        <?php
+        global $db, $params;
+        $searchquery= $_GET['searchquery'];
+        $result = $db->query("SELECT  participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit
+FROM participants
+INNER JOIN people ON participants.participantid = people.peopleid
+WHERE people.lastname LIKE '%$searchquery%' OR people.firstname LIKE '%$searchquery%' ", []);
```

Comment:
Use the query params array to put variables into an SQL statement.

Instead of:
```php
$db->query("SELECT  participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit
FROM participants
INNER JOIN people ON participants.participantid = people.peopleid
WHERE people.lastname LIKE '%$searchquery%' OR people.firstname LIKE '%$searchquery%' ", []);
```

use the array and do:

```php
$db->query("SELECT  participants.participantid, participants.dateofbirth, participants.race, people.firstname, people.lastname, people.middleinit
FROM participants
INNER JOIN people ON participants.participantid = people.peopleid
WHERE people.lastname LIKE $1 OR people.firstname LIKE $1 ", ['%'.$searchquery.'%']);
```

Reply:
Also another thought, what if I searched "John Smith"? Then it would be like `lastname LIKE '%John Smith%'` which wouldn't return any results. I do like the idea of a single search field, but the info they type should be parsed out.